VeterinaryValeriya/Menu.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VeterinaryValeriya
+{
+    class Menu
+    {
+        public struct MenuItem
+        {
+            public char Code;
+            public string Label;
+            public Action ActionToDo;
+        }
+
+        private List<MenuItem> items;
+
+        public Menu (List<MenuItem> ListOfMenuItems)
+        {
+            items = ListOfMenuItems;
+        }
+
+        public void PrintMenu ()
```

Comment:
it is better to make class Menu implement IEnumerable and print to Console outside